.travis.yml

```diff
@@ -2,20 +2,26 @@ language: python
 matrix:
   include:
     - name: Plone 5.0
-      python: 2.7
+      python: 2.7.17
       env: PLONE_VERSION=50
     - name: Plone 5.1
-      python: 2.7
+      python: 2.7.17
       env: PLONE_VERSION=51
     - name: Plone 5.2/2.7
-      python: 2.7
+      python: 2.7.17
```

Comment:
Just asking, why it is needed to specify the minor version?

Reply:
Travis was running 2.7.15 and I was thinking when I changed it was that by specifying the minor version I might have solved a bug.  It didn't work obviously but it was in there anyways so no harm no foul.